Documentation/guides/building-apps/build-items.md

```diff
@@ -174,6 +174,62 @@ installing app bundles.
 
 This build action was introduced in Xamarin.Android 11.3.
 
+## AndroidKnownDesignerAssemblies
+
+This is an ItemGroup that is used to work around issues when upgrading the old `Resource.designer.cs`
+system to the new Resource Assembly system. This can happen if an assembly is referencing a 
+`Resource.designer.cs` class from anther assembly which was built on an older framework. 
+If you see runtime errors such as 
+
+```
+System.TypeLoadException: 'Could not resolve type with token 010001d8 from typeref (expected class 'Style' in assembly '')'
+```
+
+or build errors such as
+
+```
+error NETSDK1144: Optimizing assemblies for size failed. Optimization can be disabled by setting the PublishTrimmed property to false
```

Comment:
Ditto this: i don't se this NETSDK1144 error when using the test-maui-17265 branch. :-(